x/evm/client/cli/query.go

```diff
@@ -72,17 +71,11 @@ func GetCmdAddress(queryRoute string) *cobra.Command {
 			Key:   nil,
 		}
 
-		switch {
-		case *keyRole != "" && *keyID == "":
-			keyRoleType, err := tss.KeyRoleFromSimpleStr(*keyRole)
-			if err != nil {
-				return fmt.Errorf("key role %s is not supported", *keyRole)
-			}
-			req.Key = &types.KeyAddressRequest_Role{Role: keyRoleType}
-		case *keyRole == "" && *keyID != "":
-			req.Key = &types.KeyAddressRequest_KeyID{KeyID: tss.KeyID(*keyID)}
+		switch *keyID {
+		case "":
+			req.Key = &types.KeyAddressRequest_Role{}
```

Comment:
this looks odd, is this correct?

Reply:
So keyRole isn't relevant anymore since EVM chains only have one key. I decided to keep this in the protobuf to keep things non-breaking for whoever's using the grpc call. Thought? @cgorenflo